src/fileIO.py

```diff
@@ -0,0 +1,100 @@
+from json import load
+from typing import List, Tuple
+
+from cv2 import VideoWriter, VideoWriter_fourcc
+from numpy import array
+from PIL import Image, ImageDraw, ImageFont
+
+
+class IOBase:
+    """ASCII conversion module, subclass for access to convert()"""
+
+    def __init__(self):
+        self.colours = load(open("dat/colours.json"))
```

Comment:
`data` instead of `dat`

Reply:
9668fdd7d63ecc4ffa86338cb141eb5a3282c034 and 9668fdd7d63ecc4ffa86338cb141eb5a3282c034